### ISSUE TITLE
Activity icons used inconsistently

### ISSUE THREAD

**Issue by jugglinmike** [councilforeconed/interactive-activities#144]
The "Pizza Productivity" icon is used on the project's landing page and re-used
in the heading within the activity UI itself:

![screenshot from 2015-12-08 11 16 50](https://cloud.githubusercontent.com/assets/677252/11661309/46a1e784-9d9f-11e5-8e39-4ed81d78241d.png)

All other activities are rendered *without* their associated icon from the
landing page. For example, from "The Supply and Demand of Cocoa":

![screenshot from 2015-12-08 11 17 20](https://cloud.githubusercontent.com/assets/677252/11661312/4ab1c718-9d9f-11e5-8fb1-a2521f4850b8.png)

@iamjessklein  did some excellent work with this iconography; we should make
sure they are consistently in the activity headers!

**Comment by jugglinmike** [comment]
The obvious answer here is to simply place each activity's icons within each
header as it exists today:

![screenshot from 2015-12-08 14 19 27](https://cloud.githubusercontent.com/assets/677252/11665733/e9c9fbba-9db6-11e5-9e70-cda195c9074a.png)

...but I've been thinking that the landing page establishes a vibrant color
palette, and it is a shame we don't see that reflected elsewhere:

![screenshot from 2015-12-08 14 22 12](https://cloud.githubusercontent.com/assets/677252/11665883/9b3548d2-9db7-11e5-96a8-6184180451a0.png)

So an alternative that might be nice would be to associate each activity with a
certain color, and consistently use that in the presentation of that activity
(e.g. from both the landing page *and* the activity page). In the case of "The
Supply and Demand of Cocoa", that would look something like this:

![screenshot from 2015-12-08 14 19 40](https://cloud.githubusercontent.com/assets/677252/11665886/9ec62a98-9db7-11e5-9a71-7fc95c8fba75.png)

...but of course, the "Pizza Productivity" page would not change, as it is
currently displayed with the teal background by default.

This might degrade from the cohesion of the design somewhat, but there's
precedence here: the landing page's heading is green, while the activity pages'
headings are teal.

@cbujara what do you think?

**Comment by cbujara** [comment]
I love the idea of adding the icon to the banner, and using the background color of the icon for the banner color. We may need to alter the text color for visibility for some of the darker colors. This would be a nice value-add on top of the other fixes/tweaks.

**Comment by jugglinmike** [comment]
@cbujara Yup, readability is definitely something we'd have to explicitly account for. I'll keep this feature in mind as I get through the higher-priority work.
